src/Plugin/migrate/process/SkipOnTime.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\cardinal_service_profile\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\Row;
+use Drupal\migrate_plus\Plugin\migrate\process\SkipOnValue;
+
+/**
+ * Skip the row or the process if the time is different.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "skip_on_time_compare"
+ * )
+ */
+class SkipOnTime extends SkipOnValue {
+
+  /**
+   * Transforms the value being compared from the previous process data.
+   */
+  public function transform($value, MigrateExecutableInterface $migrate_executable, Row $row, $destination_property) {
+    $original_value = strtotime($value);
+    $comparison_value = strtotime($this->configuration['value']);
```

Comment:
It's possible, but since it's populated by a developer and not the user, i'm less concerned with it. if we break it, it's on us.

Reply:
Doesn't the `$original_value` come from the migration data? Is it ever possible that data wouldn't have a valid date fed into the migration?

Reply:
Oh yeah i see what you're saying. It is possible, but most migrate process plugins expect this type of structure. there are process plugins to skip if empty. We use that skip on empty before we even get to this plugin. https://github.com/SU-HKKU/cardinal_service_profile/pull/159/files#diff-3e0ca692533e85fbdba3fb9119675ef2e52df75187b001196802f8efbf8ba523R344

Reply:
Ok that makes sense. Is adding an `if($original_value)` overkill then in this case? Just in case the `strtotime()` function on the migrated data failed.

Reply:
I think it's overkill. It won't error out or cause any issues. `strtotime('foo bar') > strtotime('now + 1day')` return false. so functionally it still works and won't blow up.

Reply:
That makes sense. If the operators won't blow it up then it all looks good to me!